views/build/Gruntfile.js

```diff
@@ -62,13 +62,23 @@ module.exports = function(grunt) {
     /*
      * Load separated configs into each extension
      */
+    const extensions = extensionHelper.getExtensions();
+
+    //MathJax fix, since it is referenced in the dependency tree, the file must be stubbed
+    if (extensions.includes('taoQtiItem')) {
+        const mathJaxEntryPoint = path.join(root, 'taoQtiItem/views/js/mathjax/MathJax.js');
```

Comment:
It won't work on windows.
```suggestion
        const mathJaxEntryPoint = path.join(root, 'taoQtiItem', 'views', 'js', 'mathjax', 'MathJax.js');
```

Reply:
Does not count, because I see hardcoded `/` at the lot of places.

Reply:
I thought `path.join` was also normalizing the paths